### ISSUE TITLE
東方天空璋 ～ Hidden Star in Four Seasons (745880)

### ISSUE THREAD

**Issue by inabahare** [ValveSoftware/Proton#2478]
# Compatibility Report
- Name of the game with compatibility issues:
- Steam AppID of the game: 745880

## System Information
- GPU: Nvidia 1060
- Driver/LLVM version 415
- Distro KDE Neon
- [Screenfetch](https://gist.github.com/inabahare/eccba884ad749101cfc2f5c58c818ee0)
- 1 4k monitor (main) and 1 1080p monitor (to the right)
- Proton version 4.2-1

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Proton log
https://gist.github.com/inabahare/915daceaa54ab3f676f0d891400b6f35

## Symptoms 
TL;DR: [This](https://youtu.be/GJMEVANXPtY)

So when I launch the game and get asked what resolution I want to play in and weather or not I want to play in fullscreen, if I press fullscreen it crashes after showing and hiding a black screen a couple of times. (The game works perfectly in Windowed mode)

## Reproduction
Launch the game in fullscreen and it should happen

**Comment by kisak-valve** [comment]
The log has some variations of:
```
err:x11settings:X11DRV_ChangeDisplaySettingsEx No matching mode found: width=1280 height=960 bpp=32 freq=60 (NoRes)
err:x11settings:X11DRV_ChangeDisplaySettingsEx No matching mode found: width=1280 height=960 bpp=32 (NoRes)
```
Before it runs into an unhandled page fault.

**Comment by AwesamLinux** [comment]
I can confirm that windowed the game works fine. But if I select fullscreen the game starts minimized (I can see the icon on the taskbar, music is playing, and the mouse pointer is trapped), alt+tabbing up the games screen everything is fine. So on my system fullscreen does not crash, but it does not launch as expected.

Distro: Linux Mint 19.1 Tessa (64 bit) standard Cinnamon desktop
GPU: Vega 56
Driver/LLVM version: Mesa 19.1.0-devel/9.0.0
Kernel version: 5.0.5-050005-generic
Proton version: 4.2-2

Included a log of where I launched the game and selected fullscreen, then brought up the games screen by alt+tabbing.
[steam-745880.log](https://github.com/ValveSoftware/Proton/files/3070201/steam-745880.log)
[system_info.txt](https://github.com/ValveSoftware/Proton/files/3070203/system_info.txt)

**Comment by AwesamLinux** [comment]
**System Information**
- GPU: RX 5700 ITX
- GPU Driver: 20.1.0-devel (git-cfa299e 2020-03-12 bionic-oibaf-ppa)
- Kernel version: 5.3.18-050318-lowlatency
- Distro version: 19.3 Tricia (64 bit) (Cinnamon)
- Proton version: 5.0-4

**Retest Compatibility Report**

With Proton 5.0-4 game works fine in Windowed mode. But it **crashes if launched in fullscreen** mode.

With older Proton-4.11-13 game launched in fullscreen mode (but minimized so the game has to be brought up)

[steam-745880-fullscreen.log](https://github.com/ValveSoftware/Proton/files/4333288/steam-745880-fullscreen.log)
[steam-745880-proton-4.11-13.log](https://github.com/ValveSoftware/Proton/files/4333289/steam-745880-proton-4.11-13.log)
[steam-745880-windowed.log](https://github.com/ValveSoftware/Proton/files/4333290/steam-745880-windowed.log)
[system_info.txt](https://github.com/ValveSoftware/Proton/files/4333291/system_info.txt)
